.github/workflows/test-matrix.yml

```diff
@@ -17,70 +17,70 @@ jobs:
         config:
         - {
             name:  "Linux Min GCC",
-            os:    "ubuntu-20.04",
-            cc:    "gcc-9",
-            cxx:   "g++-9",
-            py:    "3.8",
+            os:    "ubuntu-22.04",
+            cc:    "gcc-12",
+            cxx:   "g++-12",
+            py:    "3.9",
```

Comment:
not 3.10?

Reply:
As 22.04 has become the 'older' version to support, we test the lowest version of Python we support here.